expression-language/4.0/_index.md

```diff
@@ -1,24 +1,24 @@
 ---
-title: "Jakarta Expression Language 4.0 (milestone)"
-date: 2020-06-12
+title: "Jakarta Expression Language 4.0"
+date: 2020-07-16
 summary: "Release for Jakarta EE 9"
 ---
 Jakarta Expression Language defines an expression language for Java applications.
 
 * [Jakarta Expression Language 4.0 Release Record](https://projects.eclipse.org/projects/ee4j.el/releases/4.0.0)
   * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
-* [Jakarta Expression Language 4.0 Specification Document](./expression-langauge-spec-4.0-RC2.pdf) (PDF)
-* [Jakarta Expression Language 4.0 Specification Document](./expression-langauge-spec-4.0-RC2.html) (HTML)
+* [Jakarta Expression Language 4.0 Specification Document](./expression-language-spec-4.0.pdf) (PDF)
```

Comment:
The files need the jakarta- prefix

See PR https://github.com/eclipse-ee4j/el-ri/pull/135

Reply:
Renamed as requested.